src/main/java/cinema/controller/OrderController.java

```diff
@@ -0,0 +1,44 @@
+package cinema.controller;
+
+import cinema.dto.OrderResponseDto;
+import cinema.service.OrderService;
+import cinema.service.ShoppingCartService;
+import cinema.service.UserService;
+import cinema.service.mapper.OrderMapper;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
+    private final UserService userService;
+    private final ShoppingCartService shoppingCartService;
+    private final OrderMapper orderMapper;
+
+    public OrderController(OrderService orderService, UserService userService,
+                            ShoppingCartService shoppingCartService, OrderMapper orderMapper) {
+        this.orderService = orderService;
+        this.orderMapper = orderMapper;
+        this.shoppingCartService = shoppingCartService;
+        this.userService = userService;
+    }
+
+    @PostMapping("/complete")
+    public void completeOrder(@RequestParam Long userId) {
+        orderService.completeOrder(shoppingCartService.getByUser(userService.get(userId)));
+    }
+
+    @GetMapping
+    public List<OrderResponseDto> get(@RequestParam Long userId) {
```

Comment:
```suggestion
    public List<OrderResponseDto> getOrderHistory(@RequestParam Long userId) {
```
method names should also be informative